libs/net/template/ReportExtensions.cs

```diff
@@ -0,0 +1,107 @@
+namespace TNO.TemplateEngine;
+
+using TemplateEngine.Models.Reports;
+
+/// <summary>
+/// ReportExtensions static class, provides extension methods for report templates.
+/// </summary>
+public static class ReportExtensions
```

Comment:
This is cool.  We can pass all these functions into our Razor templates and it makes the code cleaner.